server/core/services.py

```
@@ -143,6 +143,64 @@ def get_multiple_posts(request):
+        # We assume `data` contains "text" or "image_url".
+        # Todo: handle the tags later
+        try:
+            title = data.get('title', '')
+            contentType = data.get('contentType', 'TEXT')
+            text = image_url = None
+            if contentType == 'TEXT':
+                text = data.get('text')
+            if contentType == 'IMAGE':
+                image_url = data.get('image_url')
+            try:
+                project = Project.objects.get(id=data.get('project', None))
```

Ruff (0.8.2)

Use `data.get('project')` instead of `data.get('project', None)`

Replace `data.get('project', None)` with `data.get('project')`

(SIM910)